### PR TITLE
Create EventConsumerService

### PR DESCRIPTION
**Pull request by MartinDaCosta53**
Op-codes and functions moved from CbusService.  Examples updated.

### CODE REVIEW

File: src/EventConsumerService.cpp [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  {
+    case OPC_ACON:
+    case OPC_ACON1:
+    case OPC_ACON2:
+    case OPC_ACON3:
+
+    case OPC_ACOF:
+    case OPC_ACOF1:
+    case OPC_ACOF2:
+    case OPC_ACOF3:
```
Comment (SvenRosvall) [inline src/EventConsumerService.cpp:70]: There is a discussion if these messages with data should be used with the event handler that doesn't handle that data. I am happy to leave as is for now and fix the ones with data when we know what to do with them.

File: src/EventConsumerService.cpp [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      }
+
+      return PROCESSED;
+
+    case OPC_AREQ:
+      // AREQ message - request for node state, only producer nodes
+
+      if ((msg->data[1] == highByte(module_config->nodeNum)) && (msg->data[2] == lowByte(module_config->nodeNum))) 
+      {
+        (void)(*eventhandler)(0, msg);
```
Comment (SvenRosvall) [inline src/EventConsumerService.cpp:105]: Just a note that this is the wrong behaviour. AREQ is not an incoming event. It is a request for producer to show its status. No need to fix now. 
I think it should go in the EventProducerService. Maybe put AREQ back into CbusService so we don't forget it when we create the producer service.

File: src/EventConsumerService.cpp [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+      }
+
+      return PROCESSED;
+
+    case OPC_ASRQ:
```
Comment (SvenRosvall) [inline src/EventConsumerService.cpp:110]: Oops. No implementation. Something for you to do next. :)
But again, this should be in the producer service just like AREQ.

Comment (MartinDaCosta53) [reply src/EventConsumerService.cpp:110]: I've only got in the op-codes that Duncan implemented at the moment.  He didn't implement ASRQ.
I saw Duncan's note about AREQ but the Service specs show both AREQ and ASRQ in Consumer and Producer.

**Pull request merged**